components/Plus/UserDetails.tsx

```diff
@@ -0,0 +1,87 @@
+import styled from "styled-components";
+import PlayArrowIcon from "@mui/icons-material/PlayArrow";
+import SellIcon from "@mui/icons-material/Sell";
+import CheckCircleIcon from "@mui/icons-material/CheckCircle";
+import HighlightOffIcon from "@mui/icons-material/HighlightOff";
+import PaymentIcon from "@mui/icons-material/Payment";
+import EventIcon from "@mui/icons-material/Event";
+import HourglassEmptyIcon from "@mui/icons-material/HourglassEmpty";
+
+export default function UserDetails({ user }) {
+  const Styled = {
```

Comment:
Got the concept now 👍 but please move the object above the function component (e.g. to line 9)